data/json/recipes/other/materials.json

```diff
@@ -1002,6 +1014,18 @@
     "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ] ],
     "components": [ [ [ "budget_steel_chunk", 4 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "cotton_patchwork",
+    "id_suffix": "from_rags",
```

Comment:
Is this in case the migration misses any rags?  If so maybe make a note for this to be eliminated when rags are fully obsoleted.

Reply:
Added a note to each of the remaining rag recipes.